### PR TITLE
MINOR: remove redundant argument in logging

### PR DESCRIPTION
**Pull request by chia7712**
related to #11451

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java [hunk 1]
```diff
@@ -718,7 +718,7 @@ public void handle(JoinGroupResponse joinResponse, RequestFuture<ByteBuffer> fut
                                     .setGenerationId(generation.generationId)
                                     .setAssignments(groupAssignmentList)
                     );
-            log.debug("Sending leader SyncGroup to coordinator {}: {}", this.coordinator, this.generation, requestBuilder);
+            log.debug("Sending leader SyncGroup to coordinator {}: {}", this.coordinator, this.generation);
```
Comment (dajac) [inline clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java]: I think the intent was to remove ‘generation’ in the original PR.

Comment (chia7712) [reply clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java]: oh, you are right

**Comment by showuon** [comment]
Oh, my bad! Good catch @chia7712 ! 
Please help update the PR to remove the `generation` argument. 
Thanks.

**Review by showuon — APPROVED** [review]
LGTM! Thanks for the patch!

**Review by dajac — APPROVED** [review]
LGTM

**Pull request merged**